libs/SalesforceSDKCore/SalesforceSDKCore/Classes/PushNotification/SFPushNotificationManager.m

```diff
@@ -182,7 +183,14 @@ - (BOOL)registerSalesforceNotificationsWithCompletionBlock:(SFUserAccount *)user
             [[SFPreferences currentUserLevelPreferences] setObject:strongSelf->_deviceSalesforceId forKey:kSFDeviceSalesforceId];
             [[SFPreferences currentUserLevelPreferences] synchronize];
             [SFSDKCoreLogger i:[strongSelf class] format:@"Response:%@", responseAsJson];
-            [strongSelf postPushNotificationRegistration:completionBlock];
+            [strongSelf getNotificationTypesWithRestClient:[SFRestAPI sharedInstance]
+                                                   account:[[SFUserAccountManager sharedInstance] currentUser]
```

Comment:
It's probably wouldn't make a difference in most flows but I think we should respect the user parameter from `- (BOOL)registerSalesforceNotificationsWithCompletionBlock:(SFUserAccount *)user ...` instead of using currentUser

Reply:
ah, I missed that.  I'll swap that out there.